financial-services-accelerator/components/org.wso2.financial.services.accelerator.identity.extensions/src/main/java/org/wso2/financial/services/accelerator/identity/extensions/client/registration/application/listener/AbstractApplicationUpdater.java

```diff
@@ -31,33 +31,36 @@
  */
 public abstract class AbstractApplicationUpdater {
 
-    public abstract void setOauthAppProperties(OAuthConsumerAppDTO oauthApplication, Map<String, Object> spMetaData)
+    public abstract void setOauthAppProperties(boolean isRegulatoryApp, OAuthConsumerAppDTO oauthApplication,
```

Comment:
Will the changes in this class need a change for ob3.0 users when migrating to ob4? If so,we need to track this change for migrations

Reply:
This was already available in Ob 3.0. Added to match with OB 3